docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. login with a Admin account.
+2. locate the cog
```

Comment:
Hey, a very small comment. You should start point 1 and 2 with a capital letter, just like you did with point 3, 4 and 5. This would make it look a bit cleaner imo.